source/material_model/visco_plastic.cc

```diff
@@ -425,6 +425,11 @@ namespace aspect
           else
             out.thermal_conductivities[i] = thermal_diffusivity * out.specific_heat[i] * out.densities[i];
 
+          // Replace thermal conductivities with values directly specified in the parameter, if this
+          // option was selected.
+          if (define_conductivities == true)
```

Comment:
Remove this `if` but keep the line about thermal conductivities below. Remove everything related to thermal diffusivity above.